examples/shapes.nrm

```diff
@@ -37,7 +37,8 @@ union shape
           point upper-left,
           point lower-right
       )
-    | circle (point origin, offset radius)
+    | @external-tag
```

Comment:
Should the `@external-tag` annotation be applied to tags, not union types?  Does that mean some tags could be serialized as external-tag-style while some other tags are still serialized with `"_tag"` field at a time?